src/jbms/argparse.cpp

```diff
@@ -5,11 +5,13 @@
 #include <cctype>
 #include <iostream>
 
-// Needed to determine terminal width
-// TODO: add support for Windows
-#include <sys/ioctl.h>
-#include <stdio.h>
-#include <unistd.h>
+#if defined(_WIN32) || (defined(__CYGWIN__) \
+    && !defined(_WIN32))
```

Comment:
What purpose is this `!defined(_WIN32)` condition serving?

Reply:
that's a mistake, let me update the PR.

Reply:
@jbms although that's a mistake but it's just a pre-proccessor directive we are not going to get any kind of performance hit. Do you want me to update the PR or just merge it and we will remove it later?